README.md

```diff
@@ -88,3 +88,61 @@ If the snapshot tests fail, it's generally pretty easy to tell whether it's happ
+1. A `type` - is this commit a `feat`, `fix`, `chore`, `doc`, etc.? There is a set of `type` values to choose from, but again, **only the `feat`, `fix`, `perf`, and `breaking`** `type` values will trigger a release.
+2. A `scope` - what is this commit impacting? Did you fix a bug in the `Hyperlink` component? Did you add a new feature to the `RadioButtonGroup` component? Currently, the `scope` must be lower-case and `-` separated though switching this to being `camelCase` is currently being investigated.
+3. A `subject` - provide a short description of _what_ your change is.
+  * use imperative, present tense language (so `change` not `changed` or `changes`)
+  * don't capitalize the first letter
+  * don't add a period (`.`) at the end
+  * there is a 50 character limit
+4. A `body` (optional) - add more detail about your change
+5. A `footer` (optional)
+  * > All breaking changes have to be mentioned in footer with the description of the change, justification and migration notes - [`Angular` commit message specification](https://gist.github.com/stephenparish/9941e89d80e2bc58a153#breaking-changes)
+
+#### Using `commitlint` in Paragon
+
+Paragon uses the [`commitlint`](https://github.com/marionebl/commitlint) package to lint commit messages. Paragon currently has [a `commit message` hook](https://github.com/edx/paragon/blob/master/package.json#L20) that runs `commitlint`'s, well, commit linting, on the given commit message using the configuration specified in [`commitlint.config.js`](https://github.com/edx/paragon/blob/master/commitlint.config.js). If a commit message fails linting, the commit associated with that message does not end up getting committed.
+
+`commitlint` also comes with [a helpful CLI](https://github.com/marionebl/commitlint/tree/master/@commitlint/cli) that walks one through the entire semantic commit process.
+
+![example-commitlint](https://media.giphy.com/media/3ohs7H8y9Qi7HOHWko/giphy.gif)
+
+As noted above, the only required fields are **`type`, `scope`, and `subject`**. The `body` and `footer` can be skipped through the CLI by typing `:skip`.
+
+### Pull Requests
+
+Since all pull requests should be `squashed` / `rebased` down to a single semantically-formatted commit, it is recommended that **all pull requests have a title that matches the commit message**. This way, whether you're merging a pull request, squashing and merging a pull request, or rebasing and merging a pull request, the correct commit message will be analyzed by `semantic-release`.
+
+### Merging, Building, and Releasing
+
+> When `semantic-release` is set up it will do that after every successful continuous integration build of your master branch (or any other branch you specify) and publish the new version for you
+> [`semantic-release` README](https://github.com/semantic-release/semantic-release)
```

Comment:
Formatting error here - probably need to add a `-` to indicate it's from a source